Doxyfile

```diff
@@ -964,6 +964,7 @@ RECURSIVE              = YES
 EXCLUDE                = benchmarks \
                          include/ceed/khash.h \
                          include/ceedf.h \
+                         include/ceed/jit-source \
```

Comment:
A single header that gets included in the JiT would have prevented this as well.